controller/src/board/wlan.py

```diff
@@ -0,0 +1,105 @@
+# wlan.py
+import time
+from .credentials import WLANCredentials
+import os
+
+if "rp2" in os.uname():
+    import network # type: ignore pylint: disable=E0401
+    import rp2 # type: ignore pylint: disable=E0401
+
+else:
+    from controller.tests.mpstubs import network, rp2
+
+
+# constants
+_COUNTRY = 'US'
```

Comment:
Our end users may not be in the `US`.  This constant should be able to change based on an input.

Reply:
Okay, sounds good. Where do you think the input will come from down the line? Will we be setting some initial variables before shipping the product out?